src/ethereum/frontier/vm/interpreter.py

```
@@ -86,7 +86,7 @@ def process_call(
         evm.env.state[evm.current].balance += evm.value
 
     while evm.running:
-        op = evm.code[evm.pc]
+        op = Ops(evm.code[evm.pc])
```

What happens if the instruction isn't in the enum (ex. `0xfe`)?

---

It raises a `ValueError: 254 is not a valid Ops`

---

Should we make a new error type and raise that (for invalid instructions?)

---

Sounds good. I will create a new error type for this.